HACKING.md

```diff
@@ -75,21 +75,25 @@ FAF should be available at ```http://localhost/faf/```
 Prerequisites:
 
 1. Docker, see for example [this
-guide](https://developer.fedoraproject.org/tools/docker/docker-installation.html)
+guide](https://developer.fedoraproject.org/tools/docker/docker-installation.html), or Podman - install it by running `dnf install podman`
```

Comment:
https://podman.io/getting-started/installation

And then blame them for having something broken.